packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.scss

```diff
@@ -7,18 +7,18 @@
   border: 1px solid #e7e7e7;
   border-bottom: 0;
   color: rgba(12, 12, 13, 0.8);
-  flex: 1;
-  max-width: 600px;
-  min-height: calc(100vh - #{$header-height});
-  min-width: 375px;
+  min-height: 100%;
   padding: 16px 16px 60px;
+  width: 100%;
 
   @include min-width('tablet') {
-    min-height: calc(100vh - #{$header-height-tablet});
+    border-radius: 0;
+    border-top: 0;
+    min-height: 100%;
   }
 
   @include min-width('desktop') {
-    margin: -1px 24px 0;
+    margin: -24px 0;
```

Comment:
We should avoid using negative margin like this. Instead, the `padding-top` value should be adjusted here.

![image](https://user-images.githubusercontent.com/13018240/79000918-2430e280-7b13-11ea-9627-d46972818861.png)